azure_ad_admin/plugin.spec.yaml

```
@@ -336,3 +471,26 @@ actions:
         description: Was operation successful
         type: boolean
         required: false
+triggers:
+  risk_detection:
+    title: Risk Detection
+    description: User and sign-in linked risk detections and associated information about the detection
```

Something like this. I think you can be more flowery than what I put below, but the description should be like "This is a trigger, it returns X" 

```suggestion
    description: Poll Azure for new risks
```